scripts/generate-browser-readme.js

```diff
@@ -0,0 +1,106 @@
+const fs = require("fs")
+const path = require("path")
+
+const baseImageTag = process.argv[2]
+
+const chromeVersion = process.argv.find((arg) => arg.includes("--chrome"))?.match(/(?<=--chrome=).*?(?=\s|$)/g)[0]
+const firefoxVersion = process.argv.find((arg) => arg.includes("--firefox"))?.match(/(?<=--firefox=).*?(?=\s|$)/g)[0]
+const edgeVersion = process.argv.find((arg) => arg.includes("--edge"))?.match(/(?<=--edge=).*?(?=\s|$)/g)[0]
+
+if (!baseImageTag) {
+  console.error('expected base Docker image tag like "cypress/browsers:node12.6.0-chrome77"')
+  process.exit(1)
+}
+if (!baseImageTag.startsWith("cypress/browsers:")) {
+  console.error('expected the base Docker image tag to be one of "cypress/browsers:*"')
+  console.error('but it was "%s"', baseImageTag)
+  process.exit(1)
+}
+
+if (!chromeVersion && !firefoxVersion && !edgeVersion) {
```

Comment:
I know we have some examples of using `brave` and maybe some other browsers. I didn't add any arguments or checks around any browsers beyond these 3. Do you think that is fair? Do we need to document what browsers are supported?

Reply:
That's a great suggestion. There's nothing beyond those 3 supported in this repo so that should be explicitly stated. If there is community interest, we could definitely add more.